serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,45 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+
+toc::[]
+
+If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
+
+In {ServerlessProductName} versions 1.5.0 and older, the only available Subscription Update Channel was `techpreview`.
+To upgrade to the latest version, you must update the channel to `preview-4.3`.
+
+.Prerequisites
+* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* If you have selected Manual updates, you will need to complete additional steps after updating the channel as described in this guide. The Subscription’s upgrade status will remain *Upgrading* until you review and approve its Install Plan.
+For details, see the {product-title} documentation on xref:../../operators/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[adding Operators to a cluster].
+
+.Procedure
+
+. Select the `openshift-operators` namespace in the {product-title} web console.
+. Navigate in the web console to the *Operators* → *Installed Operators* page.
+. Select the *{ServerlessOperatorName} Operator*.
+. Click *Subscription* → *Channel*.
+. In the *Change Subscription Update Channel* window, select `preview-4.3`, and click *Save*.
+. Wait until all pods have been upgraded in the `knative-serving` namespace, and the `KnativeServing` custom resource reports the latest Knative Serving version.
+
+.Verifying the upgrade
```

Comment:
A specific _Verifying_ section isn't part of mod-docs, is it possible to incorporate it into the previous procedure?

Reply:
In the mod docs guide, it does actually have Verification steps? https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

My heading is wrong here, but afaik it's valid to include this if I change it to .Verification steps instead? Is there something that contradicts this in the OpenShift manual?